app/controllers/order_imports_controller.rb

```diff
@@ -28,6 +28,14 @@ def create
     redirect_to new_facility_order_import_path
   end
 
+  def error_report
+    if order_import.error_file_present?
+      redirect_to order_import.error_file_download_url
```

Comment:
This is a little strange. When I use Fog and download this file, it's being saved as `error_report.txt`, but the filename in S3 and everything is `.csv`. Does that happen to you? It might just be a weird thing with chrome?